scripts/opentelemetry/otel_simple_setup/opentelemetry.yaml

```diff
@@ -0,0 +1,49 @@
+apiVersion: opentelemetry.io/v1alpha1
+kind: OpenTelemetryCollector
+metadata:
+  name: applicationsignals-otel
+spec:
+  image: otel/opentelemetry-collector-contrib
+  config: |
+    receivers:
+      otlp:
+        protocols:
+          grpc:
+          http:
+
+    exporters:
+      otlphttp:
+        #traces_endpoint field follows this pattern https://xray.<region>.amazonaws.com/v1/traces
+        traces_endpoint: https://xray.us-east-1.amazonaws.com/v1/traces 
+        auth:
+          authenticator: sigv4auth
+          
+    extensions:
+      sigv4auth:
+        region: us-east-1
```

Comment:
Will any of the new OTel Collector configuration examples be used in an actual script?
If so, can `us-east-1` be replaced by a placeholder? If not, I'd suggest renaming `scripts` to `example_configurations`.

Reply:
I think it is ok to use us-east-1 by default, want to provide a working example.